packages/loader/driver-utils/src/blobAggregationStorage.ts

```diff
@@ -185,17 +165,20 @@ export class BlobAggregationStorage extends SnapshotExtractor implements IDocume
     static wrap(
         storage: IDocumentStorageService,
         logger: ITelemetryLogger,
-        allowPacking = gatesAllowPacking(),
+        allowPacking?: boolean,
         packingLevel = 2,
     ) {
         if (storage instanceof BlobAggregationStorage) {
             return storage;
         }
+        const mc = loggerToMonitoringContext(logger);
+        const realAllowPackaging = mc.config.getBoolean("FluidAggregateBlobs") ??  allowPacking ?? false;
```

Comment:
namespace?

Reply:
Also, I think this changes the semantics of the default value of `allowPacking`.  Previously, config was only checked if the arg was not supplied.  Now the config is the authority and the param is the fallback.